templates/viewer/passage_footer.html

```diff
@@ -0,0 +1,6 @@
+{% if "elexicon" in passage_data.objectId %}
```

Comment:
Is this file any different than the copy of it in the main template? If not, why have you recopied it here?